Debugger/Debugger.m

```diff
@@ -92,7 +92,7 @@
 
 	populateDebuggerInformation[sowedAssignments,sowedMessages];
 
-	If[MatchQ[return,$Aborted[]],
+	If[MatchQ[return,$Aborted],
```

Comment:
Do we still need this conditional? Can we just throw all caught messages?

Trying to talk about a scenario where AbortOnMessage is set to False, the all messages are caught as Failures, and the reapReturn is some real return so we ended up quieting all messages

Reply:
Ok I changed that